### PR TITLE
Changed variable renderOnlineUsers to renderOnline in soluion seed.

### PR DESCRIPTION
**Pull request by ayushkr7**
In the challenge seed renderOnline variable has been used and referenced but in the solution seed the variable has been changed to renderOnlineUser so, changed renderOnlineUsers to renderOnline variable.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]
LGTM! Congrats on creating your first pull request! 🥳 

This will require one more approval from a member of the team before it is able to be merged, however. :)

**Review by RandellDawson — APPROVED** [review]

**Pull request merged**
